tools/gbench/util.py

```diff
@@ -131,7 +131,7 @@ def benchmark_wanted(benchmark):
         if benchmark_filter is None:
             return True
         name = benchmark.get("run_name", None) or benchmark["name"]
-        if re.search(benchmark_filter, name):
+        if re.search(benchmark_filter, name) is not None:
```

Comment:
how about just

`return re.search(benchmark_filter, name) is not None`

Reply:
Good call! Done in [6eece2b](https://github.com/google/benchmark/pull/1740/commits/6eece2bc2465facb62b69ab1285ee7634be23cab).